README.md

```diff
@@ -12,21 +12,28 @@ The architecture is pluggable and the tooling is flexible to make it applicable
 - [DNS as code - Tools for managing DNS across multiple providers](#dns-as-code---tools-for-managing-dns-across-multiple-providers)
 - [Table of Contents](#table-of-contents)
 - [Getting started](#getting-started)
-  - [Workspace](#workspace)
-  - [Config](#config)
-  - [Noop](#noop)
-  - [Making changes](#making-changes)
-  - [Workflow](#workflow)
-  - [Bootstrapping config files](#bootstrapping-config-files)
-- [Supported providers](#supported-providers)
-    - [Notes](#notes)
+  * [Workspace](#workspace)
+    + [Installing a specific commit SHA](#installing-a-specific-commit-sha)
+  * [Config](#config)
+  * [Noop](#noop)
+  * [Making changes](#making-changes)
+  * [Workflow](#workflow)
+  * [Bootstrapping config files](#bootstrapping-config-files)
+- [Providers](#providers)
+  * [Updating to use extracted providers](#updating-to-use-extracted-providers)
+- [Sources](#sources)
+    + [Notes](#notes)
+- [Compatibilty & Compliance](#compatibilty---compliance)
+  * [`lenient`](#-lenient-)
+  * [`strict_supports` (Work In Progress)](#-strict-supports---work-in-progress-)
+  * [Configuring `strict_supports`](#configuring--strict-supports-)
 - [Custom Sources and Providers](#custom-sources-and-providers)
 - [Other Uses](#other-uses)
-  - [Syncing between providers](#syncing-between-providers)
-  - [Dynamic sources](#dynamic-sources)
+  * [Syncing between providers](#syncing-between-providers)
+  * [Dynamic sources](#dynamic-sources)
 - [Contributing](#contributing)
 - [Getting help](#getting-help)
-- [Related Projects & Resources](#related-projects--resources)
+- [Related Projects & Resources](#related-projects---resources)
```

Comment:
Click-and-jump is not working on this anchor link.

Reply:
Hrm, used one of those auto-generator deals, but I guess it didn't support the &. Will rejigger things to get them happy when I get a chance to sit down with it.